### PR TITLE
add console alerting

### PR DESCRIPTION
**Pull request by russellkemmit**
### Short Summary
Add Console dispatcher for when the other dispatcher types are not available

### Context
This updates the dispatch module to include traits so that we can define functions and methods needed for a dispatcher to work properly. 

### Is this a Breaking Change?
No!

### CODE REVIEW

File: src/alerts/dispatch.rs [hunk 1]
```diff
@@ -1,40 +1,43 @@
 use anyhow::{Context, Result};
+use async_trait::async_trait;
 use scouter::utils::types::FeatureAlerts;
 use serde_json::{json, Value};
 use std::env;
 
+trait DispatchHelpers {
```
Comment (thorrester) [inline src/alerts/dispatch.rs:7]: 👍 🔥

File: src/alerts/dispatch.rs [hunk 2]
```diff
@@ -112,12 +96,60 @@ impl OpsGenieAlertDispatcher {
     }
 }
 
+#[derive(Debug)]
+pub struct ConsoleAlertDispatcher;
+
+#[async_trait]
+impl Dispatch for ConsoleAlertDispatcher {
```
Comment (thorrester) [inline src/alerts/dispatch.rs:103]: we'll want to add an additional test for `ConsoleAlertDispatcher`

File: src/alerts/dispatch.rs [hunk 3]
```diff
@@ -112,12 +96,60 @@ impl OpsGenieAlertDispatcher {
...
+
+impl DispatchHelpers for ConsoleAlertDispatcher {}
+
+impl ConsoleAlertDispatcher {
+    fn send_alerts(alert_description: &str, model_name: &str) {
+        println!(
+            "{} is experiencing drift. \n{}",
+            model_name, alert_description
+        );
+    }
```
Comment (thorrester) [inline src/alerts/dispatch.rs:122]: 🥳

File: src/alerts/dispatch.rs (hunk 2 above)
Comment (russellkemmit) [reply src/alerts/dispatch.rs:109]: Done

**Pull request merged**